pinot-controller/src/test/java/org/apache/pinot/controller/helix/ControllerTest.java

```diff
@@ -152,6 +154,21 @@ public static ControllerTest getInstance() {
     return DEFAULT_INSTANCE;
   }
 
+  public List<String> createPhysicalTables(List<String> physicalTableNames)
```

Comment:
PhysicalTableNames should have type. Dont pass in hybrid table names. Or rename the function to contain `hybridTable`

Reply:
Refactored it to `createHybridTables`.